UnityProjects/MRTKDevTemplate/ProjectSettings/ProjectSettings.asset

```diff
@@ -156,8 +156,8 @@ PlayerSettings:
   androidSupportedAspectRatio: 1
   androidMaxAspectRatio: 2.1
   applicationIdentifier:
-    Android: com.Microsoft.MRTK3Sample
-    Standalone: com.Microsoft.MRTK3Sample
+    Android: com.MixedRealityToolkitOrganization.MRTK3Sample
+    Standalone: com.Mixed-Reality-Toolkit-Organization.MRTK3-Sample
```

Comment:
org.MixedRealityToolkit

Reply:
Overrode the auto-genned strings to match our domain